run_performance_tests.sh

```
@@ -0,0 +1,266 @@
+  echo -e "${RED}❌ Flutter is not properly installed${NC}"
+  exit 1
+}
+
+# Get dependencies
+echo -e "${BLUE}📦 Getting dependencies...${NC}"
+flutter pub get
+
+# Clean previous results
+if [ "$CI_MODE" = true ]; then
+  echo -e "${YELLOW}🧹 Cleaning previous results...${NC}"
+  rm -rf "$RESULTS_DIR"/*
```

Shellcheck (0.10.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)